lib/index.js

```diff
@@ -112,9 +112,18 @@ module.exports = (secp256k1) => {
     publicKeyCreate (seckey, compressed = true, output) {
       isUint8Array('private key', seckey, 32)
       isCompressed(compressed)
+
+      let res
       output = getAssertedOutput(output, compressed ? 33 : 65)
+      if (!bitcoinerlab) {
+        res = secp256k1.publicKeyCreate(output, seckey)
+      } else {
+        const bRes = secp256k1.publicKeyCreate(seckey, compressed)
+        res = bRes.res
+        output = bRes.output
+      }
```

Comment:
why can't bitcoinerlab `secp256k1` write to `obj` like the elliptic wrapper does and return the same output (a status code)?

Reply:
That's a good point. Will try to improve API so that there are fewer changes in `index.js`. Maybe even no changes. Except maybe `ecdsaSign` changes